### DISCUSSION TITLE
Improvement of the graph

### DISCUSSION THREAD

**Discussion by ndeblauw** [spatie/flareapp.io-roadmap#32] [Ideas]
Hi guys,

I'm a huge fan of the spike graph that you added to the overview page. This is really intuitive and easy to grasp an overview! Well done!!!

I would suggest to take it even one small step further, and not only use it for historical data, but also add today's errors to the graph. I imagine that this is more difficult to cache, and thus causes a heavier load on the server, but it really would add a lot of usability. Visual scanning of where to look for an issue in a multi-project environment will get a lot faster.

Below a quick visualisation of how I imagine it could look like.
![image](https://user-images.githubusercontent.com/4344869/115526405-474ab200-a290-11eb-9a5c-c2d51ec595be.png)

**Comment by AlexVanderbist** [comment]
Interesting! I think this might even be considered a bug as you'd expect it would work like this. I'll add it to our internal issue tracker :)

**Comment by tomasvanrijsse** [comment]
It seems that this request has been implemented.
I can see the current day in the chart on the overview page.

**Reply by AlexVanderbist** [reply]
That's correct! We fixed this last week

**Reply by ndeblauw** [reply]
@AlexVanderbist (and team): and it is very appreciated! But it is not yet fully working, it seems. Please take the comments below as feedback on improving a function, and not as nagging of an annoying client. It's intended 100% as a helping hand.

I just re-invoked an (existing) error, and even though it is caught correctly (in the latest column), it doesn't show as a "today's spike"
![image](https://user-images.githubusercontent.com/4344869/138671126-8be00950-45ee-475e-b3a4-f59f36509287.png)

Then I thought this might was because I reinvoked an existing error. So I invoked a "fresh" new error. Also this is correctly shown in the "latest" column, but doesn't show up as a today's spike.
![image](https://user-images.githubusercontent.com/4344869/138671397-00c6bf4b-8e23-463d-8b0a-92ceeee3272d.png)

**Reply by AlexVanderbist** [reply]
Hey @ndeblauw, thanks for the feedback, I've recently noticed this as well. This is due to caching (1 hour right now) and we're currently looking into getting this to work realtime without having a performance impact. I'll update this thread once I know more.
